src/components/shared/__tests__/ProfileModal.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { MockedProvider, MockedResponse } from '@apollo/react-testing';
+import React, { ReactElement } from 'react';
+import Friend from '../../screen/Friend';
+import { QUERY_FRIENDS } from '../../../graphql/queries';
+import { createTestElement } from '../../../../test/testUtils';
+import { render } from '@testing-library/react-native';
+
+describe('[ProfileModal] rendering test', () => {
+  const setup = (mocks?: Array<MockedResponse>): ReactElement => {
+    // Insert mock at index of 0
+    mocks && mocks.unshift({
+      request: {
+        query: QUERY_FRIENDS,
+      },
+      result: {
+        data: {
+          friends: [{
+            id: 'aa11',
+            photoURL: '',
+            name: 'testName',
+          }],
+        },
+      },
+    });
+
+    return createTestElement(
+      <MockedProvider mocks={mocks} addTypename={false}>
+        <Friend />
+      </MockedProvider>);
+  };
+
+  it('Render without crashing', () => {
+    const component = render(setup());
```

Comment:
I ran `yarn test` this file, throw error. so I tried to add this below, error disappeared.
so consider put this code.
```suggestion
  await wait(() => {
    const component = render(setup());
    expect(component).toMatchSnapshot();
  });
```

Reply:
I don't know why an error occurs when I didn't warp await wait(() => { ... }) do you know why is it?
```
  console.error node_modules/react-native/Libraries/YellowBox/YellowBox.js:63
    Warning: An update to Screen inside a test was not wrapped in act(...).
```

Reply:
I've have curious about this situation, and I googled then found a relative reference.
- [github issue: this is just a warning](https://github.com/testing-library/native-testing-library/issues/91#issuecomment-564625613)

so only way to remove this warning, we have to find something to configure.
I think many people are suffering from this situation. here is [other github issues](https://github.com/testing-library/native-testing-library/issues/25)